radix-engine-common/src/types/entity_type.rs

```diff
@@ -44,6 +44,9 @@ pub enum EntityType {
     /// A global native access controller entity (195 in decimal). Gives Bech32 prefix: `c` followed by one of `v`, `d`, `w` or `0`.
     GlobalAccessController = 0b11000011, //---------- 11000 => c, 011xx => vdw0 (011 = access controller)
 
+    /// A global native pool entity (196 in decimal). Gives Bech32 prefix: `c` followed by one of `s`, `3`, `j` or `n`.
+    GlobalSingleResourcePool = 0b11000100, //---------- 11000 => c, 100xx => s3jn (100 = pool)
```

Comment:
We will need to have three entity types for the pools. One for the single, two, and n pools.

This is because of the `to_typed_object_substate_key_internal` function in the [`typed_substate_layout.rs`](https://github.com/radixdlt/radixdlt-scrypto/blob/develop/radix-engine-queries/src/typed_substate_layout.rs#L184) which seems to be going for the abstraction that the layout of a substate could be determined based on the entity type of the node that it belongs to.